content/blogs/gui-setup-part2-partitioning.md

```diff
@@ -0,0 +1,104 @@
+
+{{< gallery >}}
+{{< figure link="/img/blogs/hbelusca_gui-setup/usetup_partition_list.png" src="/img/blogs/hbelusca_gui-setup/usetup_partition_list.png" caption="USETUP - Partition list">}}
+{{< /gallery >}}
+
+While the creation/deletion operations made from this screen are not immediately applied to the disk, they get applied _as soon_ as the user selects a suitable partition, or an eligible empty space, for installing ReactOS.
+There, the new partition layout is written to the disk and the installation continues, with the choice of a filesystem for formatting the selected or newly-created partitions.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/hbelusca_gui-setup/usetup_partition_filesystem.png" src="/img/blogs/hbelusca_gui-setup/usetup_partition_filesystem.png" caption="USETUP - Partition filesystem selection">}}
+{{< /gallery >}}
+
+The actual partition formatting takes place after the filesystem selection.
+Once all the newly-created partitions (and perhaps the selected one) have been formatted, they are checked for filesystem defects.
+This is particularly true for the selected [_boot partition_](http://jdebp.info/FGA/boot-and-system-volumes.html) for ReactOS installation
+(as well as the [_system partition_](https://en.wikipedia.org/wiki/System_partition_and_boot_partition) where the bootloader is installed; these can be the same usually on BIOS-based PCs, but can otherwise be different).
+
+After all these steps have been performed, the installer inquires the user for the ReactOS installation directory, and where to install the bootloader.
+The ReactOS installation proper then begins, with file copy and saving the settings,
+then continues after system restart with the 2nd-stage installer.
+
+
+### Workflow change in the GUI-mode setup
+
+Albeit the installation workflow used by the text-mode USETUP _might_ make sense, from a UI perspective (each action is done from a different screen; only forward progress is allowed, and once an action has been chosen, it cannot be undone by e.g. going back to previous screens), the GUI-mode setup changes some of these assumptions.
+The wizard-style of the GUI setup allows going back and forth within the different pages.
+
+Its partitioning page shows a minimal interface, similar to the text-mode one, but more reminiscent of other GUI partitioning software.
+
+Changing the disk partition layout and formatting new or existing partitions can be considered to be a sensitive operation -- existing partitions or filesystems can be destroyed after all!
+The user then may feel more comfortable performing such operations virtually, viewing the resulting changes, knowing they haven't yet been actually applied, until his/her final choice has been made.
+This is why, for example, 3rd-party partitioning software allow to stash (queue) all these changes, allowing for a careful review before actually applying them.
```

Comment:
```suggestion
This is why, for example, 3rd-party partitioning software allow to stash (queue) all these changes, allowing to review them carefully before actually applying.
```

Reply:
@cbialorucki : Is "allowing to" OK in this case?

Reply:
This entire sentence is difficult. Maybe try something like this:
```suggestion
This is why, for example, 3rd-party partitioning software allows the user to stash (queue) all these changes, allowing him or her to carefully review the changes before applying them.
```

Reply:
Suggestion for the suggestion:
```suggestion
This is why, for example, 3rd-party partitioning software allows users to stash (queue) all these changes, allowing them to carefully review the changes before applying them.
```